lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -26,7 +66,7 @@ def template_path
   end
 
   def vendored_modules_path
-    File.expand_path(File.join(__FILE__, '..', '..', '..', '..', '..', 'vendor','DSCResources')).gsub(/\//,'\\')
+    "#{@top_level_path}\\vendor\\DSCResources"
```

Comment:
FYI this is going away pretty soon as part of #53 since files are not pluginsync'd from this directory.